README.md

```diff
@@ -19,3 +19,21 @@ Creates PDF with Quarkus extensions contain given dependency (quarkus-vertx, qua
 - Usage:`get-extensions.sh DEPENDENCY_NAME [OUTPUT_FILE_NAME].pdf` 
 - Example: `./get-extensions.sh quarkus-vertx extensions-vertx.pdf`
 - Result will be written by default to the PDF document `Quarkus-extensions-with-*.pdf` or to the file with your custom name
+
+### artifact-version-diff
+
+Create the report of different version between two Qaurkus version. This can be used for upstream vs RHBQ or upstream vs upstream.
+
+- Preparation:
+  - `git clone https://github.com/quarkusio/quarkus.git`
+  - `git checkout <tag>` tag should be version which will be base for comparison e.g. 3.8.3
+  - `mvn versions:compare-dependencies -DremotePom=com.redhat.quarkus.platform:quarkus-bom:<version_to_compare> -Dmaven.repo.local="<path_to_local_repository>" -DreportOutputFile=<name_of_output_file>`
+- Generation
+  - `mvn clean install exec:java -Dpath-to-quarkus-repository=<path_to_quarkus_repo> -Dversion-plugin.report-output-file=<name_of_output_file_from_version_plugin> -DskipTests`
```

Comment:
Adding further to the comment before, maybe the utility should be executing the preparation stage itself.

Reply:
I'll look on this if we can do this and if it will be easier or harder.

Reply:
Updated with better explanation and example commands